scripts/fuzz_shell.js

```diff
@@ -47,9 +47,25 @@ if (!binary) {
 // passed a final parameter in the form of "exports:X,Y,Z" then we call
 // specifically the exports X, Y, and Z.
 var exportsToCall;
-if (argv.length > 0 && argv[argv.length - 1].startsWith('exports:')) {
-  exportsToCall = argv[argv.length - 1].substr('exports:'.length).split(',');
-  argv.pop();
+
+// Passing --fuzz-split makes us treat the two input files as split off
+// from a single one, and we will run them as that single file (ignoring extra
+// exports from the second one, and from wasm-split itself). This allows us to
+// get the same behavior from split modules as before the split.
+var fuzzSplit = false;
+
+while (argv.length > 0) {
+  var last = argv[argv.length - 1];
+  if (last.startsWith('exports:')) {
+    exportsToCall = last.substr('exports:'.length).split(',');
+    argv.pop();
+    continue;
+  } else if (last.startsWith('--fuzz-split')) {
+    fuzzSplit = true;
+    argv.pop();
+    continue;
+  }
```

Comment:
Can we generalize this to support passing `--fuzz-split` in any position rather than only after the modules are given? I can see myself messing this up in the future.

Reply:
Done.